pallets/frequency-tx-payment/src/lib.rs

```diff
@@ -274,7 +273,7 @@ impl<T: Config> Pallet<T> {
 
 	/// Compute the length portion of a fee by invoking the configured `LengthToFee` impl.
 	pub fn length_to_fee(length: u32) -> BalanceOf<T> {
-		T::LengthToFee::weight_to_fee(&Weight::from_ref_time(length as u64))
+		T::LengthToFee::weight_to_fee(&Weight::from_parts(length as u64, 0))
```

Comment:
I will check if we need to do anything with the proof size, or if always setting it to 0 is sufficient.